<ui version="4.0" >
 <class>resetDialog</class>
 <widget class="QDialog" name="resetDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Reset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="optionsBox" >
     <property name="title" >
      <string>Please select reset mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QRadioButton" name="softOption" >
          <property name="text" >
           <string>Soft </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hardOption" >
          <property name="text" >
           <string>Hard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="noneOption" >
          <property name="text" >
           <string>None</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>resetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>289</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel" >
     <x>251</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>resetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>358</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>339</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
